sdk/python/feast/repo_config.py

```diff
@@ -291,11 +294,16 @@ def offline_store(self):
     def auth_config(self):
         if not self._auth:
             if isinstance(self.auth, Dict):
-                self._auth = get_auth_config_from_type(self.auth.get("type"))(
-                    **self.auth
+                is_oidc_client = (
+                    self.auth.get("type") == AuthType.OIDC.value
```

Comment:
probably miss to check other client fields after the recent review

Reply:
Done.